404-server/OWNERS

```diff
@@ -0,0 +1,4 @@
+assignees:
+- wernight
```

Comment:
Ah, right. Will replace. I think it did not choose him because his changes are over 6 months old. We should probably alter the logic to use more than just the commits in the past 6 months if that is often the case.